developer-docs-site/docs/cli-tools/aptos-cli-tool/use-aptos-cli.md

```diff
@@ -536,6 +536,20 @@ Test result: OK. Total tests: 2; passed: 2; failed: 0
 }
 ```
 
+### Proving Move
+
+The `aptos` CLI can be used to run [Move Prover](https://github.com/move-language/move/tree/main/language/move-prover) which is a formal verification tool for the Move language. The below example proves the `hello_prover` package in [move-examples](../../aptos-move/move-examples/).
```

Comment:
Link broken for move-examples. Correct link is `[move-examples](https://github.com/aptos-labs/aptos-core/tree/main/aptos-move/move-examples/hello_prover)`.

Reply:
Got it. Fixed.